website/views/pages.py

```diff
@@ -58,6 +59,10 @@ def honeycomb(request):
     context = {}
     all_honeycomb_posts = HoneycombPost.objects.all()
     context['all_honeycomb_posts'] = all_honeycomb_posts
+
+    context['all_youtube_videos'] = get_youtube_videos(
+        'UCHnEuCRDGFOR5cfEo0nD3pw', 100)
```

Comment:
actually I am not sure yet if we will have a single youtube channel or multiple. May be it will be better to make it editable from the admin panel?